### DISCUSSION TITLE
blog/statsandr-is-2-years-old/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#95] [General]
# blog/statsandr-is-2-years-old/

Learn how to track page views, sessions, users engagement and more Google Analytics data in R with the {googleAnayticsR} R package

https://statsandr.com/blog/statsandr-is-2-years-old/

**Comment by shivako58** [comment]
Hi good morning! I read the blog and found helpful. Thank you for sharing.

**Reply by AntoineSoetewey** [reply]
Glad you liked it !

_Disclaimer: I removed the link in your comment because a link to a real estate services company is totally unrelated to the topic of this article. Thanks for your understanding._
